CommonUI/Sources/FeatureComponent/MovieRow.swift

```diff
@@ -15,12 +15,12 @@ public struct MovieRow: View {
                 if let image = phase.image {
                     image
                         .resizable()
-                        .frame(width: 80, height: 100)
+                        .aspectRatio(contentMode: .fit)
                         .cornerRadius(4)
                 } else if phase.error != nil {
                     Image(systemName: "film")
                         .resizable()
-                        .frame(width: 80, height: 100)
+                        .aspectRatio(contentMode: .fit)
```

Comment:
the `AsyncImage` can be refactored a bit to allow more images to appear on the screen

by adding
```
.frame(maxWidth: 80)
```
![Simulator Screenshot - iPhone 15 Plus - 2024-02-20 at 15 42 29](https://github.com/AndrewBrehmAtDialexa/MovieAppMicro/assets/97919323/f21bef50-adb2-448b-8d43-98421709ce7b)

as well as move the related image attributes outside to the larger `AsyncImage` element (see below)

```
AsyncImage(url: URL(string: movie.posterUrl)) { phase in
                if let image = phase.image {
                    image
                        .resizable()
                } else if phase.error != nil {
                    Image(systemName: "film")
                        .resizable()
                        .foregroundStyle(Color.secondaryLight)
                } else {
                    ProgressView()
                }
            }
            .cornerRadius(4)
            .aspectRatio(contentMode: .fit)
            .frame(maxWidth: 80)
            .padding(10)
           
```